index.js

```diff
@@ -1,7 +1,53 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Julio";
+console.log("The driver's name is " + hacker1);
+
+let hacker2 = "Christian";
+console.log("The navigator's name is " + hacker2);
+
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log("The driver has the longest name, it has " + hacker1.length + " characters.");
+  } else if (hacker1.length < hacker2.length) {
+    console.log("It seems that the navigator has the longest name, it has " + hacker2.length + " characters.");
+  } else {
+    console.log("Wow, you both have equally long names, " + hacker1.length + " characters!");
```

Comment:
```suggestion
    console.log("Wow, you both have equally long names, " + hacker1.length + " characters!");
```

Esto esta perfect pero si tuvieras que usar la interpolacion como lo harias??

Reply:
Asi jejeje!
    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);